recipes/tafra/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tafra" %}
+{% set version = "1.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a0affd16c6c47123cb2832fcd8dc11d0e6b8a02752d3368f63a8060b2fd431f0
+
+build:
+  number: 3
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - numpy >=1.17
+    - typing-extensions
```

Comment:
`host` contains the dependencies needed for building the packages (i.e. running setup.py) not the dependencies for using it. In almost all cases `python` and `pip` are enough:

```suggestion
```

Reply:
In this case since the `setup.py` imports the package, the `run` dependencies will need to be listed under `host`.

Reply:
I've left these in for now, will remove the import of the package in the `setup.py` in the next version.